mods/voidmulti.sc2mod/base.sc2data/gamedata/unitdata.xml

```diff
@@ -1762,6 +1943,9 @@
             <LayoutButtons Face="BurrowUp" Type="AbilCmd" AbilCmd="BurrowInfestorUp,Execute" Row="2" Column="4"/>
             <LayoutButtons Face="BurrowUp" Type="AbilCmd" AbilCmd="BurrowUltraliskUp,Execute" Row="2" Column="4"/>
         </CardLayouts>
+        <Radius value="0.875"/>
+        <InnerRadius value="0.6875"/>
+        <MinimapRadius value="0.875"/>
```

Comment:
Ultralisk: Reduced size by 12.5%

These are the values for the Ultralisk's burrowed version. They are identical to the unburrowed Ultralisk as they should be